src/client/editor/utils/sortEntities.js

```diff
@@ -1,30 +1,30 @@
 export default function sortEntities(entitiesObj) {
   // Sort entities by insertion order
-  const sortedEntityNames = [];
```

Comment:
Changes to this file are just renaming things from name -> id to better reflect the data structure.